hackathon_site/event/jinja2/event/landing.html

```diff
@@ -32,7 +32,7 @@ <h4 class="header s12">Subtitle</h4>
                     <h5>Location</h5>
                     <h5>
                         {# Update this logic if your event ends in a different month or year #}
-                        {{ localtime(event_start_date).strftime("%B %-d") }}-{{ localtime(event_end_date).strftime("%-d, %Y") }}
+                        {{  event_start_date | formatDateTime("%B %-d") }}-{{ event_end_date | formatDateTime("%B %-d") }}
```

Comment:
Add a space between the '-'. Currently it looks like this on my end. (MacOS)
<img width="644" alt="CleanShot 2022-05-29 at 15 12 13@2x" src="https://user-images.githubusercontent.com/36619969/170887629-00f5d14a-bdbf-4397-8107-248939d9618f.png">

Reply:
Whoops! Thanks for the catch, I accidentally put in the wrong format after making the filter. It's been fixed!